### DISCUSSION TITLE
Effect: fade not working with slidesPerView greater than one

### DISCUSSION THREAD

**Discussion by shmdhussain** [nolimits4web/swiper#5653] [Q&A] [answered]
I like to fade multiple slides usine slidesPerView: 2,

In the code sandbox https://codesandbox.io/s/swiper-responsive-breakpoints-forked-qcr71c?file=/index.html:2236-2249 , i tried to have **fade effect with slidesPerView: 2, greater than one** 

In this case swiper is showing only one slide, as I browse the repo issues found this issue https://github.com/nolimits4web/swiper/issues/2062 , there it is mentioned that fade effect works only with slidesPerView: 1. 

Is this restriction still there for fadeEffect ?

I am unable to find this restriction on Documentation. Thanks in advance for any help

**Comment by nolimits4web** [comment, marked as answer]
> Is this restriction still there for fadeEffect ?

Yes

**Reply by shmdhussain** [reply]
Thanks for this wonderful swiper and super quick response.

For my curiosity, why this restriction has been there

I like to have this feature, any scenario is stopping from having this feature?

I love to have this feature here. anyhow thanks again for all your wonderful work and making this open source.
